utils_cv/tracking/references/fairmot/tracker/multitracker.py

```diff
@@ -190,6 +189,7 @@ def __init__(self, opt, frame_rate=30):
         self.std = np.array(opt.std, dtype=np.float32).reshape(1, 1, 3)
 
         self.kalman_filter = KalmanFilter()
+        BaseTrack._count = 0 # EDITED
```

Comment:
What does this do / why is this needed?

Reply:
So before this was added, if you had the notebook open and you ran predict() multiple times (let's say twice, for this example), the following would happen: 
first run:
predict() -> 3 tracks with track_ids: [0, 1, 2]
second run:
predict() -> 4 tracks with track_ids: [3, 4, 5, 6]

This is not the desired result because we could be running predict() on two different datasets, just with the same TrackingLearner object.

This was happening because of how the `nextid` method was implemented in BaseTrack in the source code; I guess because the code was designed to be run on the command line.

Reply:
Great, thanks for the explanation.